firefox_puppeteer/ui/windows.py

```diff
@@ -188,7 +188,7 @@ def __eq__(self, other):
 
     @property
     def closed(self):
-        """Returns closed state of the chrome window
+        """Returns closed state of the chrome window.
```

Comment:
I tried to add periods to sentences and verb phrases consistently throughout and otherwise tried to use periods consistently within each file, aiming for a minimal review.